public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/comment-meta.php

```diff
@@ -143,6 +143,16 @@ function get_feedback_meta_field_schema( $context = 'all', $key = '' ) {
 			'default'           => false,
 			'required'          => false,
 		),
+		'speaker_notified' => array(
+			'sft_context'       => array( 'update' ),
```

Comment:
This meta value is editable by the speaker via the API, like `helpful` — did you intend that? (I'm not sure if it matters, but it seemed unexpected)

Reply:
I was aware of that when I added it, but it seems unlikely that it would ever be set through the API in a way that we don't intend. I guess we could add another value for `sft_context`. Maybe `internal` or something that indicates it's only used programmatically?

Reply:
`internal` sounds good 👍

Reply:
I guess another option would be to just not include it in the field schema, since, as mentioned, it's only for internal usage...

Reply:
Decided to go with `internal`: 6c6207b5